support-services/src/main/scala/com/gu/support/redemption/gifting/GiftCodeValidator.scala

```diff
@@ -0,0 +1,35 @@
+package com.gu.support.redemption.gifting
+
+import com.gu.support.redemption.gifting.GiftCodeValidator.{expirationTimeInMonths, getSubscriptionState}
+import com.gu.support.redemption.{CodeAlreadyUsed, CodeExpired, CodeNotFound, CodeRedeemedInThisRequest, CodeValidationResult, ValidGiftCode}
+import com.gu.support.redemptions.RedemptionCode
+import com.gu.support.zuora.api.response.SubscriptionRedemptionQueryResponse
+import com.gu.zuora.{ZuoraGiftLookupService, ZuoraService}
+import org.joda.time.LocalDate
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object GiftCodeValidator {
+  val expirationTimeInMonths = 12
+
+  def getSubscriptionState(existingSub: SubscriptionRedemptionQueryResponse, requestId: String): CodeValidationResult =
```

Comment:
```suggestion
  def getSubscriptionState(existingSub: SubscriptionRedemptionQueryResponse, requestId: Option[String]): CodeValidationResult =
```
then we don't need to pass in an empty string for nothing